### PR TITLE
Add Ldodson orlando file

### LINKED ISSUE #19: Create a new file for ldodson Orlando
- [ ] Create a branch
- [ ] Add commits
- [ ] Open PR
- [ ] Discuss
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by LDodson**
closes #19

### CODE REVIEW

**Comment by svatasoiumms** [comment]
good job

**Comment by James-Mulcahy** [comment]
All Your Base Are Belong to Us

File: ldodson-orlando.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Orlando, Fl
+
+### Recomendations
+- Try Flippers Pizza
```
Comment (James-Mulcahy) [inline ldodson-orlando.md:4]: Mmmmm pizza

Comment (svatasoiumms) [inline ldodson-orlando.md:4]: pizza, pizza :+1:

**Review by James-Mulcahy — APPROVED** [review]

**Pull request merged**